modules/kogito-swf-builder/module.yaml

```
@@ -4,28 +4,31 @@ version: "2.0.0-snapshot"
 description: "Kogito Serverless Workflow builder with required extensions"
 
 envs:
+  - name: QUARKUS_EXTENSIONS
+    example: 'quarkus-kubernetes,kogito-quarkus-serverless-workflow,kogito-addons-quarkus-knative-eventing'
+    description: To add extension to your application
   - name: PROJECT_GROUP_ID
     value: "org.acme"
-    description: Please do not change
+    description: Please do not change. To change only if you plan to use the ${KOGITO_HOME}/launch/create-app.sh
```

```suggestion
    description: Change only if you plan to use the ${KOGITO_HOME}/launch/create-app.sh script
```
is there any example on how to use it?